### ISSUE TITLE
 get static facial expression after training in my dataset

### ISSUE THREAD

**Issue by jyangliu** [EvelynFan/FaceFormer#22]
After training FaceFormer with my dataset,I find that wave2vec2.0 output a constant feature for all audio,so i got a static facial expression. I add Periodic Positional Encoding in the training time. How to fix that?

**Comment by jyangliu** [comment]
I solve this problem by seting learning_rate=10e-5 because my own dataset is between [0,1] while vocast dataset is between [-0.1,0.1],so i get a to large learning rate.

**Comment by ujjawalcse** [comment]
@jyangliu @zlinao @EvelynFan , Me too getting the same. 
Also, couldn't be fixed with the Learning rate tuning.
Any suggestions?

**Comment by Echo-jyt** [comment]
> I solve this problem by seting learning_rate=10e-5 because my own dataset is between [0,1] while vocast dataset is between [-0.1,0.1],so i get a to large learning rate.
您好，请问您能说说你的数据集的详细信息吗？我现在用的我自己的准备的数据集，48个视频，每个视频的人不同。其中40个训练数据，3个验证数据，5个测试数据。但是我现在的结果就是人脸的嘴巴张幅很小，嘴巴几乎不怎么动，很奇怪。
